client/package.json

```diff
@@ -102,6 +102,7 @@
   "dependencies": {
     "@bugsnag/js": "^7.16.2",
     "@types/react-dom": "^18.0.1",
+    "autosuggest-highlight": "^3.3.4",
```

Comment:
@mark-prins any complains to include a new package?

is it correct to include it here? why not in common where it is used?

Reply:
slightly reluctant, but ok.
I would have just written something to do the highlight, as it's a fairly simple task. I would have missed the ignoring of diacritics though, so this is probably best.
There's quite an overhead with maintaining packages, in addition to the increased bundle size.. so you're right to query. We should only add packages if there's good reason to.

I would add it to common, rather than the root. If it is only used in one package, best to just include it in that.